### ISSUE TITLE
Request: Prioritize side exercises over legacy core exercises

### ISSUE THREAD

**Issue by F3PiX** [exercism/exercism#4426]
Since the introduction of the Mentor on Request, the number of side exercises is very low (in Ruby, in the last count there were 2). We all agree that the side exercises are important for the student's learning experience, and I think especially on the level of the core exercise they were unlocked by. As in: as soon as possible after they did the core exercise.  

To give the side exercises the importance they deserve and to maximise the learning results, could we boost the sides to appear after the new core but before the legacy core exercises?  Given the low numbers, I think that's not unfair to the legacy students.

**Comment by ErikSchierboom** [comment]
I believe this is already the case. My mentor dashboard first shows core exercise, then side exercises and then core (legacy) exercises.

**Comment by iHiD** [comment]
Yeah - it should already be the case, but I will verify before closing!

**Comment by F3PiX** [comment]
🤔 
Here's the bottom part of my mentor dashboard.
There are mentor requests for side exercises, but they don't show up before the legacy core's.

   

![schermafbeelding
 2018-10-17 om 21 03 29](https://user-images.githubusercontent.com/6314687/47110151-5c25d500-d250-11e8-9c65-3d1339cd335a.png)

**Comment by iHiD** [comment]
Ah. Legacy. I read Independent.

I think legacy core are more important than side, as these are just users who happened to solve the exercise in v1, but now want to progress in v2. As opposed to users who are in independent mode.

Thoughts?

**Comment by F3PiX** [comment]
I also think that legacy core are more important, but even so I pledge to have side exercises before legacy core in the current queue. For 2 reasons:
- there are currently only a few side exercises (in Ruby) with mentoring requests, and those are probably by the most active students, because they had to be here in the last few days. 
- there's a lot of repetition in the mentoring queue, and for mentors it may be a nice surprise to see something other than Hamming and Grains every once and a while. 

With the low number of side exercises, I would expect that there's maybe one or two per day, so it won't delay the core legacy much. While side exercises probably have to wait months before we managed to clear out the legacy core and finally get to them.

**Comment by F3PiX** [comment]
@ihid And one more thing: with the restructuring of the Ruby track, I feel it's a big advantage if side exercises are mentored before the student progressed into higher levels. 

Heads up: I haven't seen even one side exercise showing up in my mentor queue since the change to Mentor on Request.

**Comment by rpottsoh** [comment]
> Heads up: I haven't seen even one side exercise showing up in my mentor queue since the change to Mentor on Request.

Me neither in the Delphi track.  That just might mean that none of those students has requested mentoring for their side submissions.  But I have no way to know that.

**Comment by F3PiX** [comment]
If you're determinant:  on the mentor dashboard, you can filter 'Next solutions to mentor' by exercise  :-))
Or request a mentoring on one of your own side exercises and ask other mentors if it shows up in their queue. :-)

**Comment by rpottsoh** [comment]
> request a mentoring on one of your own side exercises and ask other mentors if it shows up in their queue. :-)

I have sort of just done this.  I have a student that has completed all the available exercises and has taken to adding exercises to the track.  🥇  I am able to communicate with this person pretty easily and so have asked him if he has opted-in on mentoring any of his unmentored side exercise submissions.  I think I have tried your first suggestion, but will look again.  Delphi is a pretty sleepy track, but was seeing a nice uptick in activity over the past 2-3 months.  With the change all the visible activity went away....

**Comment by iHiD** [comment]
I'm really against punishing someone and not letting them progress on the track just because they happened to solve the exercise on July 13th not July 13th. Unless someone can explain to me why we should do this I'm going to close this as "won't fix" :)

**Comment by iHiD** [comment]
OK - cool. Thanks :)
